lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -237,6 +248,7 @@
         space_guarantee: none
         policy: default
         percent_snapshot_space: 60
+        qos_policy_group: max_performance_gold
```

Comment:
can you also add qos_adaptive_policy_group as an example  (here or some new example) ?

Reply:
added a new example with adaptive qos policy